scripts/masterCron.ts

```diff
@@ -58,6 +63,110 @@ const getTopOfPeriod = async (provider: any, period: number) => {
   return topOfPeriod;
 };
 
+async function getStrikePrice(
```

Comment:
copying from logic in vault lifecycle [commitAndClose](https://github.com/ribbon-finance/ribbon-v2/blob/bd9b69f2b68132cea8d418709c00acab57e815e0/contracts/libraries/VaultLifecycle.sol#L53)